files/zuul-tests.yaml

```
@@ -12,6 +12,16 @@
           - podman
         state: present
       become: true
+    - name: Create ~/.local/share/
+      file:
+        path: ~/.local/share/
+        state: directory
+        recurse: yes
+    - name: Create ~/.local/share/containers
+      file:
+        path: ~/.local/share/containers
+        state: directory
+        setype: data_home_t
```

Could the two be squashed? AFAIK, `podman` creates `~/.local/share/containers` when runs the first time. It `~/.local/share` has the right context, newly created subdirs should also be OK.

```suggestion
    # Fix the SELinux context for podman
    - name: Create ~/.local/share/
      file:
        path: ~/.local/share/
        state: directory
        recurse: yes
        setype: data_home_t
```

---

I think that `~/.local/share` should not have selinux type `data_home_t` as should have the `~/.local/share/containers` dir.

But maybe I am wrong?

---

I ended up reading _gentoo wiki_ about this… imho you can use `setype: data_home_t` with ~/.local/share` too

---

Gentoo is the best :)

Sorry for the confusion. I made the recommendation above b/c I have `data_home_t` on `~/.local/share` in my home directory. So I've assumed that this is the default there.